File: countvowels/python/count_vowels.py
#!/usr/bin/python3
from collections import Counter

def countvowels(data):
    return sum([v for k,v in Counter(data).items() if k in 'aeiouAEIOU'] )

def countvowels2(data):
    return sum ( data.count(i) for i in 'aeiouAEIOU' )


if __name__ == "__main__":
    print(countvowels("hello"))

    assert(countvowels('abracadabra') == countvowels2('abracadabra'))

File: countvowels/csharp/countvowels.cs
using System;
using System.Linq;


class VowelCounter {

    public static int countVowels ( string args )
    {
        return args.ToList().Count(ch => "aeiouAEIOU".IndexOf(ch) != -1);
    }

    public static void Main(string[] args)
    {
        string inputStr = "hello";
        Console.WriteLine("Counting the vowels in hello: " + countVowels(inputStr));

    }


}
